scripts/systems.py

```diff
@@ -157,24 +156,22 @@
 """,
 }
 
-# NOTE: Here is assumed that `gpu2ranks_slurm_hip` is in PATH!
+# NOTE: Here is assumed that `gpu2ranks_slurm_cuda` is in PATH!
 #       modify "Run command" if it is not the case.
-cscs["hohgant-amdgpu"] = {
+cscs["clariden-nvgpu"] = {
```

Comment:
Would it make sense to do
```
cscs["clariden-nvgpu"] = cscs["hohgant-nvgpu"]
```
and same for `*-amdgpu` (assuming I see correctly that they actually are the same)?

There may be good reasons for keeping them separate as well (being able to easily make one-off changes to one or the other system) so not at all insisting on this.